v6/fetchLists/index.js

```
@@ -0,0 +1,218 @@
+    headers: {
+      'User-Agent': v4()
+    }
+  });
+  const data = await response.json();
+  const tokens = Object.values(data.tokens).reduce((currVal, t) => {
+    if (t && t.address !== '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee') {
+      t.contract_address = t.address.toLowerCase();
+      t.icon = IMAGE_PROXY + t.logoURI;
+      t.icon_png = IMAGE_PROXY + encodeURIComponent(t.logoURI);
+      t.network = network;
+      delete t.logoURI;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)